README.md

```diff
@@ -1,345 +1,77 @@
-# rivet-react npm package
+[Rivet-react](https://github.com/indiana-university/rivet-react) is a library of React components that implement the [Rivet Software Design System](https://rivet.uits.iu.edu). The components are implemented in [TypeScript](http://www.typescriptlang.org/) and can be used by any Javascript application.
+
+Rivet-react resources:
+* [Component Documentation and Style Guide](https://indiana-university.github.io/rivet-react/)
+* [GitHub repo](https://github.com/indiana-university/rivet-react)
+* [NPM package](https://www.npmjs.com/package/rivet-react)
+* [Travis CI](https://travis-ci.org/indiana-university/rivet-react)
 
 | Branch | Build | Code Coverage |
 | ------ | ----- | ------------- |
 | master | [![Build Status](https://travis-ci.org/indiana-university/rivet-react.svg?branch=master)](https://travis-ci.org/indiana-university/rivet-react) | [![codecov](https://codecov.io/gh/indiana-university/rivet-react/branch/master/graph/badge.svg)](https://codecov.io/gh/indiana-university/rivet-react/branch/master) |
 | develop | [![Build Status](https://travis-ci.org/indiana-university/rivet-react.svg?branch=develop)](https://travis-ci.org/indiana-university/rivet-react) | [![codecov](https://codecov.io/gh/indiana-university/rivet-react/branch/develop/graph/badge.svg)](https://codecov.io/gh/indiana-university/rivet-react/branch/develop) |
 
+## Installation
+
+To install the rivet-react package from NPM,
+
+```shell
+npm install rivet-react
+```
 
-Published to [https://www.npmjs.com/package/rivet-react](https://www.npmjs.com/package/rivet-react)
+Or with yarn,
 
-## Installing the Library from 
+```shell
+yarn add rivet-react
+```
 
-With npm:
+Once installed you can use the components in your project. The Rivet styles are included as a dependency of *rivet-react* but they are not bundled into the components, so you'll need to import them as well.
 
-    npm install rivet-react
+```typescript
+import * as React from 'react'
+import 'rivet-uits/css/rivet.min.css'
+import { Alert } from 'rivet-react'
 
-Or with yarn:
+<Alert variant="info">A very important message for you!</Alert>
+``` 
 
-    yarn add rivet-react
+Check out the [Component Documentation and Style Guide](https://indiana-university.github.io/rivet-react/) for comprehensive component documentation and examples.
 
-## Building Locally
+## Contributing
 
-You can build with either npm or yarn.
+[Rivet-react](https://github.com/indiana-university/rivet-react/) is open source and maintained by members of the [Indiana University](https://github.com/indiana-university) organization on GitHub. If you have questions about the library or encounter problems please [file an issue](https://github.com/indiana-university/rivet-react/issues). If you wish to contributed, [pull requests](https://help.github.com/articles/about-pull-requests/) are welcome!
```

Comment:
Typo here: `contributed` should be `contribute`

Reply:
Fixed